### PR TITLE
Bug 1822269 - add androidx.activity-compose dependency

### PR DESCRIPTION
**Pull request by HarrisonOg**
### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Breaking Changes**: If this is a breaking Android Components change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-apk-{fenix,focus,klar}-debug` task you're interested in.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1822269

### CODE REVIEW

File: fenix/buildSrc/src/main/java/FenixDependencies.kt [hunk 1]
```diff
@@ -88,6 +89,7 @@ object FenixDependencies {
     const val androidx_profileinstaller = "androidx.profileinstaller:profileinstaller:${FenixVersions.androidx_profileinstaller}"
     const val androidx_biometric = "androidx.biometric:biometric:${FenixVersions.androidx_biometric}"
     const val androidx_fragment = "androidx.fragment:fragment-ktx:${FenixVersions.androidx_fragment}"
+    const val androidx_activity_compose = "androidx.activity:activity-compose:${FenixVersions.androidx_activity_compose}"
```
Comment (rahulsainani) [inline fenix/buildSrc/src/main/java/FenixDependencies.kt:92]: I think we should explicitly add `androidx.activity-ktx` dependency as well as that gets [pulled in with this](https://android.googlesource.com/platform/frameworks/support/+/refs/heads/androidx-main/activity/activity-compose/build.gradle#32) and we haven't explicitly declared that dependency in the project yet.

**Review by rahulsainani — APPROVED** [review]
Looks good! 👍  Left one comment about `androidx.activity-ktx` dependency.

File: fenix/buildSrc/src/main/java/FenixDependencies.kt (hunk 1 above)
Comment (HarrisonOg) [reply fenix/buildSrc/src/main/java/FenixDependencies.kt:93]: @rahulsainani added the dependency for `androidx.activity-ktx`

Comment (rahulsainani) [reply fenix/buildSrc/src/main/java/FenixDependencies.kt:93]: thanks!

**Pull request merged**